tools/binlog/overview.md

```diff
@@ -0,0 +1,59 @@
+---
+title: TiDB-Binlog Cluster 版本用户文档
+category: tools
+---
+
+# TiDB-Binlog 简介
+
+TiDB-Binlog 的整体架构：
+
+![TiDB-Binlog 架构](../../media/tidb_binlog_cluster_architecture.png)
```

Comment:
/media/tidb_binlog_cluster_architecture.png seems a ineffective link

Reply:
@ericsyh The `../../media/tidb_binlog_cluster_architecture.png` format is also right. 

Our website supports the `/media/tidb_binlog_cluster_architecture.png` format. I changed it because later I'll rename all the docs-cn documents and folders. There will be multiple levels of directories (for SEO). This is to make it easier to guarantee the correctness of one link, because this format is always right in whatever directories.